client_plugin/drivers/shared/kvstore/etcdops/etcdops.go

```diff
@@ -521,13 +522,14 @@ func addrToEtcdClient(addr string) (*etcdClient.Client, error) {
 func (e *EtcdKVS) List(prefix string) ([]string, error) {
 	var keys []string
 
-	etcd := e.createEtcdClient()
-	if etcd == nil {
-		return nil, fmt.Errorf(etcdClientCreateError)
+	etcdAPI := e.createEtcdClient()
```

Comment:
The above change doesn't look correct.
clientv3 doesn't mean too much.
So I kept where etcdClient still as etcdClient, but changed etcdAPI to client since it's limited inside each function.